content/en/blog/intro-gvisor-falco/index.md

```diff
@@ -43,25 +46,25 @@ Once a message related to a syscall is received, Falco unpacks it and creates th
 
 ![](/blog/intro-gvisor-falco/images/intro-gvisor-falco-03.png)
 
-## Setup gVisor sandbox monitoring with Falco
+## Setup gVisor Docker sandbox monitoring with Falco
 
-First, [install **Falco**](https://falco.org/docs/getting-started/installation/) 0.32.1 or above and [install the **gVisor runsc tool**](https://gvisor.dev/docs/user_guide/install/) 20220704.0 or above.
+First, [install **Falco**](https://falco.org/docs/getting-started/installation/) 0.33.1 or above and [install the **gVisor runsc tool**](https://gvisor.dev/docs/user_guide/install/) 20221122.0 or above.
 
 You can check the version by running:
 ```bash
 falco --version
 ```
-... which needs to report 0.32.1 or above and:
+... which needs to report 0.33.1 or above and:
 ```bash
 runsc --version
 ```
-... which needs to report `release-20220704.0` or above.
+... which needs to report `release-20221122.0` or above.
 
-gVisor needs to be configured to send events to Falco. Download the appropriate configuration file:
+gVisor needs to be configured to send events to Falco. Generate the appropriate configuration file:
 
 ```bash
-$ sudo wget -O /etc/docker/runsc_falco_config.json \
-    https://falco.org/blog/intro-gvisor-falco/assets/config.json
+$ falco --gvisor-generate-config > /tmp/runsc_falco_config.json
```

Comment:
The `assets/` directory wouldn't be necessary anymore. I think the commit should also remove it.

Reply:
Sure, done!